src/modules/authenticationModules.js

```diff
@@ -0,0 +1,58 @@
+// import JtockAuth from "j-tockauth";
+import axios from 'axios'
+import store from '../state/stores/configureStore'
+
+// const auth = new JtockAuth({
+//   host: "http://localhost:3000",
+//   prefixUrl: "/api",
+//   authUrl: "admin_auth"
+// });
+
+// const signingIn = async (event) => {
+//   event.preventDefault()
+//   let credentials = {
+//     email: event.target.email.value,
+//     password: event.target.password.value
+//   }
+//   try {
+//     await auth.signIn(credentials.email, credentials.password)
+//     debugger
+//     store.dispatch({ type: "AUTHENTICATE" })
+//   } catch (error) {
+//     debugger
+//     return error.response ? error.response.data.errors : error.message
```

Comment:
Our auth path is "/admin_auth", whereas the default for J-tockauth is just "/auth". 
But when I try to change it, its other functionality errors out (the signInUrl becomes undefined e.g.)